Gopkg.toml

```diff
@@ -45,3 +45,12 @@
 [[constraint]]
   branch = "release-1.8"
   name = "k8s.io/kubernetes"
+
+[[constraint]]
```

Comment:
the new dependencies should be reflected in NOTICE.md

I think @hoeltcl should review changes to dependencies to ensure we don't add dependencies which come with a license we don't want and we cannot consume according to internal standards. We should agree on a process here and document it in https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md

Reply:
I've added the new Google Go SDK description to the `NOTICE.md` file.
The update of other dependency is respecting the constraints in the `Gopkg.toml` file, thus that should be fine.
I don't think somebody should manually review each PR regarding new dependencies/dependencies with disallowed licenses/... We should rather invest into some automation which does these things automatically. Already now with only a few number of projects the double maintenance of `Gopkg.toml` and `NOTICE.md` is very tedious...

Reply:
yes, that's a good idea. I'll also discuss this with @hoeltcl